docs/notifications.json

```
@@ -4,14 +4,14 @@
     "title": "<b>Check out Base UI today</b> 💥",
     "text": "Love Material UI, but don't need Material Design? Try Base UI, the new \"unstyled\" alternative. <a style=\"color: inherit;\" data-ga-event-category=\"Blog\" data-ga-event-action=\"notification\" data-ga-event-label=\"introducing-mui-base\" href=\"/blog/introducing-mui-base/\">Read more in this announcement</a>."
   },
-  {
-    "id": 74,
-    "title": "<b>Introducing MUI X v6.0.0</b>",
-    "text": "<a style=\"color: inherit;\" data-ga-event-category=\"Blog\" data-ga-event-action=\"notification\" data-ga-event-label=\"mui-x-v6\" href=\"/blog/mui-x-v6/\">Explore</a> what's new and what's next in the new stable version of the advanced components."
-  },
   {
     "id": 75,
     "title": "<b>Try the new time picker UI now</b>",
     "text": "The new time-picking experience is designed for desktops, and it's available on the <a style=\"color: inherit;\" data-ga-event-category=\"Announcement\" data-ga-event-action=\"notification\" data-ga-event-label=\"mui-x-v6\" href=\"https://mui.com/x/react-date-pickers/time-picker/\">Time Picker</a> as well as a <a style=\"color: inherit;\" data-ga-event-category=\"Announcement\" data-ga-event-action=\"notification\" data-ga-event-label=\"mui-x-v6\" href=\"https://mui.com/x/react-date-pickers/digital-clock/\">standalone component</a>."
+  },
+  {
+    "id": 76,
+    "title": "<b>Charts are finally here!</b>",
```

I think that this would better manage the expectation of developers, the current label makes it feel like it's in GA.

```suggestion
    "title": "<b>Introducing Charts, in alpha</b>",
```

---

```suggestion
    "title": "<b>Unveiling Charts: Alpha release is live</b>",
```